### ISSUE TITLE
Bayesian Json test fails on opening file

### ISSUE THREAD

**Issue by Katka92** [redhat-developer/che-functional-tests#483]
This is an intermittent issue, sometime the project tree is not opened and the test fails to open the file:
```
[ERROR] prepareProjectFile(com.redhat.che.functional.tests.BayesianPackageJson)  Time elapsed: 262.812 s  <<< FAILURE!
org.openqa.selenium.TimeoutException: 
Expected condition failed: waiting for visibility of element located by By.xpath: //div[@path='/nodejs-hello-world/package.json']/div
```
This error is followed by error in ```closeFiles``` method.

**Comment by Katka92** [comment]
I've investigated the issue and was not able to reproduce it. I would say it can be a problem with loosing a focus of an element - e.g. when pop-up is opened or the right tab for git appears. From investigations I've made, it seems that ```Repository``` tab is opened and steal focus. [1] After discussion with @ibuziuk we decided not to invest time on writing workaround and investing time here as far as it fails in ~10% of runs. I'll keep monitoring that issue. 

[1] 
![image](https://user-images.githubusercontent.com/29278800/56891530-15c1f600-6a7d-11e9-95ca-9659a2050262.png)
